source/win-assets/SnippetsTab.vue

```diff
@@ -302,11 +305,15 @@ function openSnippetsDirectory (): void {
 }
 
 #snippets-container {
-  padding: 10px;
+  padding: 0px 10px;
```

Comment:
…can't we define one class to use? Even if we have repeated code then, we would still see that the divs do the same thing